docs/usage/types/custom.md

```diff
@@ -188,11 +188,13 @@ If the Generic class that you are using as a sub-type has a classmethod
 Because you can declare validators that receive the current `field`, you can extract
 the `sub_fields` (from the generic class type parameters) and validate data with them.
 
-```py test="xfail - what do we do with generic custom types"
-from typing import Generic, TypeVar
+```py
+from typing import Any, Generic, TypeVar, get_args, get_origin
+
+from pydantic_core import CoreSchema, core_schema
 
 from pydantic import BaseModel, ValidationError
-from pydantic.fields import ModelField
+from pydantic.annotated import GetCoreSchemaHandler
```

Comment:
Any chance we could change this import path before the beta release? If you prefer it I'll shut up but it just feels like `annotated` is a very confusing module name for it.

My preference would be just in `from pydantic import GetCoreSchemaHandler` but could do `pydantic.handlers` or something. Since the module only has `__all__ = 'GetJsonSchemaHandler', 'GetCoreSchemaHandler'` it seems `handlers` is a better name than `annotated`.

Reply:
Also let's leave this for another change